StreamVideoSwiftUITests/CallView/LayoutComponents/DominantSpeakerLayoutComponent_Tests.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamVideo
+@testable import StreamVideoSwiftUI
+import SnapshotTesting
+import XCTest
+import Foundation
+
+@MainActor
+final class DominantSpeakerLayoutComponent_Tests: StreamVideoUITestCase {
```

Comment:
The dog generated images are too big (>100KB). We should try to reduce them or use the other ones, so we don't increase the repo size.